packages/client/utils/getContentGroup.ts

```diff
@@ -0,0 +1,24 @@
+export default function getContentGroup(pathname: string) {
```

Comment:
This is copied from the current setting in [UA](https://analytics.google.com/analytics/web/#/a74109115w112182245p117099558/admin/content-groupings/m-content-contentGroupingTable.rowShow=10&m-content-contentGroupingTable.rowStart=0&m-content-contentGroupingTable.sortColumnId=dateModified&m-content-contentGroupingTable.sortDescending=true)